src/lib.rs

```diff
@@ -77,6 +80,64 @@ pub enum Error {
 
 type Result<T> = std::result::Result<T, Error>;
 
+#[no_mangle]
+pub extern "C" fn Sanity() -> i32 {
+    0
+}
+
+const EXISTING: Mutex<Option<Connection>> = Mutex::new(None);
```

Comment:
It's better practice to return and ask for a void* pointer in C bindings instead of having a single static object.